specification.markdown

```diff
@@ -2,9 +2,9 @@
 
 EBML, short for Extensible Binary Meta Language, specifies a binary and octet (byte) aligned format inspired by the principle of XML (a framework for structuring data).
 
-The goal of this document is to define a generic, binary, space-efficient format that can be used to define more complex formats using an EBML Schema. EBML is used by the multimedia container [Matroska](https://github.com/Matroska-Org/matroska-specification/). The applicability of EBML for other use cases is beyond the scope of this document.
+The goal of this document is to define a generic, binary, space-efficient format that can be used to define more complex formats using an EBML Schema. EBML is used by the multimedia container, Matroska [@!Matroska]. The applicability of EBML for other use cases is beyond the scope of this document.
```

Comment:
The `!` makes it normative. I think `[@Matroska]` is the format we want to use here.

Reply:
Thx. I changed the `!` to `?`.